config/source_input.go

```diff
@@ -20,13 +20,13 @@ type SourceInputConfig interface {
 // EmptySourceInputConfig sets un-configured source inputs with a non-null
 // value
 func EmptySourceInputConfig() SourceInputConfig {
-	return &NoSourceInputConfig{}
+	return &NoMonitorConfig{}
 }
 
 // isSourceInputEmpty returns true if the provided SourceInputConfig `c` is
 // of type NoSourceInputConfig
```

Comment:
change comment to be type `NoMonitorConfig`

Reply:
Ah, thanks for catching!